ui/client/components/templater/TextAnnotater.js

```diff
@@ -87,6 +87,7 @@ const Mark = ({
   const displayInnerContent = () => (
     <span
       className={savedHover ? classes.darkerMark : classes.mark}
+      data-testid="annotation-inner-contents"
```

Comment:
This is testid and not test, as it's not a custom `cy.get(''[data-test=...)` and instead it is a built-in plugin: `cy.findByTestId(...)`. We'll consolidate in follow-up work as needed.